internal/app/api/handler/response.go

```diff
@@ -16,7 +16,7 @@ type (
 )
 
 // sendErrorResp generates and sends error response
-func sendErrorResp(c fiber.Ctx, statusCode int, errorMessage string) error {
+func sendErrorResponse(c fiber.Ctx, statusCode int, errorMessage string) error {
```

Comment:
This func suppose to be named as sendErrorResp, please pull latest branch commit.